docs/source/getting-started.rst

```diff
@@ -1,48 +1,246 @@
-2. Install `Terraform <https://www.terraform.io/intro/getting-started/install.html>`_
-3. Download StreamAlert:
+2. Install `Terraform <https://www.terraform.io/intro/getting-started/install.html>`_ v0.11.X:
 
 .. code-block:: bash
 
-  $ git clone --branch stable https://github.com/airbnb/streamalert.git
+  brew install terraform  # MacOS Homebrew
+  terraform --version     # Must be v0.11.X
 
-4. Install and activate Virtualenv:
+3. Install `virtualenv <https://virtualenv.pypa.io/en/stable/installation/>`_:
 
 .. code-block:: bash
 
-  $ cd streamalert
-  $ pip install --user virtualenv
-  $ virtualenv -p python2.7 venv
-  $ source venv/bin/activate
+  pip install --user virtualenv
+  virtualenv --version
 
-5. Install dependencies:
+4. If you're on a Linux system, you may need to install the Python development libraries:
 
 .. code-block:: bash
 
-  $ pip install -r requirements.txt
+  sudo apt install python-dev    # Debian
+  sudo yum install python-devel  # CentOS/RHEL
 
+Download StreamAlert
+--------------------
 
-.. note:: If you run into issues with ``psutil`` installation, make sure you have the ``python-dev`` dependencies installed.  `debian: sudo apt install python-dev``, `CentOS/RHEL: sudo yum install python-devel``
+1. Clone the latest stable release of StreamAlert:
 
+.. code-block:: bash
+
+  git clone --branch stable https://github.com/airbnb/streamalert.git
+
+2. Create and activate a virtual environment:
+
+.. code-block:: bash
+
+  cd streamalert
+  virtualenv -p python2.7 venv
+  source venv/bin/activate
+
+3. Install the StreamAlert requirements:
+
+.. code-block:: bash
+
+  pip install -r requirements.txt
+
+4. Run unit tests to make sure everything is installed correctly:
+
+.. code-block:: bash
+
+  tests/scripts/unit_tests.sh
+
+Set AWS Credentials
+-------------------
+
+1. Create an AWS account and an IAM user with permissions for at least the following services:
+
+* Athena
+* CloudTrail
+* CloudWatch Events and Logs
+* DyanmoDB
+* IAM
+* Kinesis Firehose and Streams
+* KMS
+* Lambda
+* S3
+* SNS
+* SQS
+
+2. Configure your AWS credentials
+
+.. code-block:: bash
+
+  pip install --user awscli
+  aws configure
+
+Deploy
+------
+
+1. Set basic StreamAlert configuration options:
+
+.. code-block:: bash
+
+  ./manage.py configure aws_account_id 123412341234  # Replace with your 12-digit AWS account ID
+  ./manage.py configure prefix company_team          # Choose a unique name prefix
+
+2. Build the StreamAlert infrastructure:
+
+.. code-block:: bash
+
+  ./manage.py terraform init
+
+There will be multiple Terraform prompts, type "yes" at each one to continue.
+
+3. At this point, StreamAlert is up and running! You can, for example, see the S3 buckets
+that were automatically created:
+
+.. code-block:: bash
+
+  aws s3 ls | grep streamalert
+
+You can also login to the AWS web console and see StreamAlert's CloudWatch logs, Lambda functions, etc.
+
+Live Test
```

Comment:
👍  This is awesome! Using SNS to test StreamAlert is very convenient and easy. Thank you for adding this!

Reply:
No problem! Thanks for the review